README.md

```diff
@@ -68,10 +65,17 @@ The expected format of the service account key is a **json** with the following
 }
 ```
 
-3. Configure the service account key and private key for authentication in the SDK by following one of the alternatives below:
-   - setting the fiels in the provider block: `service_account_key` or `service_account_key_path`, `private_key` or `private_key_path`
-   - setting environment variables: `STACKIT_SERVICE_ACCOUNT_KEY_PATH` and `STACKIT_PRIVATE_KEY_PATH`
-   - setting `STACKIT_SERVICE_ACCOUNT_KEY_PATH` and `STACKIT_PRIVATE_KEY_PATH` in the credentials file (see above)
+3. Configure the service account key for authentication in the SDK by following one of the alternatives below:
+
+   - setting the fiels in the provider block: `service_account_key` or `service_account_key_path`
+   - setting the environment variable: `STACKIT_SERVICE_ACCOUNT_KEY_PATH`
+   - setting `STACKIT_SERVICE_ACCOUNT_KEY_PATH` in the credentials file (see above)
+
+4. **If you have provided your own RSA key-pair**, you can set it the same way (it will take precedence over the private key included in the service account key, if present):
+
+   - setting the fiels in the provider block: `private_key` or `private_key_path`
+   - setting the environment variable: `STACKIT_PRIVATE_KEY_PATH`
+   - setting `STACKIT_PRIVATE_KEY_PATH` in the credentials file (see above)

```

Comment:
I feel like this shouldn't be a 4th point of the previous steps, it could either be:
- A different section (e.g. after `## Key Flow` having something like `### Generation a new SA Key` and `### Using your own RSA key pair`)
- Or (I think this one is easier to read) make it something like a note. E.g.:

```suggestion
> **If you have provided your own RSA key-pair**, you can set it the same way (it will take precedence over the private key included in the service account key, if present):

   - setting the fiels in the provider block: `private_key` or `private_key_path`
   - setting the environment variable: `STACKIT_PRIVATE_KEY_PATH`
   - setting `STACKIT_PRIVATE_KEY_PATH` in the credentials file (see above)
```

Reply:
Also I am a bit confused by the phrasing, especially `**If you have provided your own RSA key-pair**, you can set it the same way`

Do you mean that if the user wants to use their own key-pair and provide the private key separately, they can follow a similar approach as to setting the service account key (provider config, env. variable or configuration file)?

Reply:
Adapted it and also rephrased